components/Author.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+
+import AuthorCollection from '../interfaces/AuthorCollection'
+import { MarkdownFileWithUrl } from '../pages/authors'
+
+import ContentCardList from './ContentCardList'
+import PageLayout from './PageLayout'
+
+export interface Props {
+ validAuthor: AuthorCollection
+ filteredRecordsWithUrl: MarkdownFileWithUrl[]
+}
+
+const Author: React.FunctionComponent<Props> = props => {
+  const authorName = props.validAuthor.name
+  return (
+      <PageLayout documentTitle={` ${authorName}`} appendSiteTitle={true}>
```

Comment:
Let's give it a title:
```suggestion
      <PageLayout documentTitle={`Resources by ${authorName}`} appendSiteTitle={true}>
```